backend/scripts/monitor_ai_processing.py

```
@@ -0,0 +1,338 @@
+#!/usr/bin/env python3
+"""
+AI处理实时监控脚本
+监控 summary 和 key_points 的处理状态
+"""
+
+import asyncio
+import json
```

Ruff (0.11.9)

`json` imported but unused

Remove unused import: `json`

(F401)